<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemPage</class>
 <widget class="QWidget" name="SystemPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1011</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Store_tab">
    <attribute name="title">
     <string>店面管理</string>
    </attribute>
    <widget class="QToolBox" name="Store_toolBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>981</width>
       <height>541</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Store_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="label">
       <string>店面管理</string>
      </attribute>
      <widget class="QTableView" name="StoreManagement_tableView">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>41</y>
         <width>951</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>379</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Search_pushButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Add_pushButton">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Delete_pushButton">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Amend_pushButton">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Member_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="label">
       <string>会员管理</string>
      </attribute>
      <widget class="QTableView" name="MemberManagement_tableView">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>41</y>
         <width>951</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>379</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Search_pushButton_2">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Add_pushButton_2">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Delete_pushButton_2">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Amend_pushButton_2">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Charge_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="label">
       <string>费用管理</string>
      </attribute>
      <widget class="QTableView" name="ChargeManagement_tableView">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>41</y>
         <width>951</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>10</y>
         <width>379</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="Search_pushButton_3">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Add_pushButton_3">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Delete_pushButton_3">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Amend_pushButton_3">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Member_tab">
    <attribute name="title">
     <string>会员模块</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Order_tab">
    <attribute name="title">
     <string>订单管理</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Laundry_tab">
    <attribute name="title">
     <string>洗衣店模块</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Settings_tab">
    <attribute name="title">
     <string>设置</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>391</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="logout_pushButton">
        <property name="text">
         <string>注销</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Exit_pushButton">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
